readthedocs/projects/urls/private.py

```diff
@@ -103,6 +103,10 @@
     url(r'^(?P<project_slug>[-\w]+)/advertising/$',
         ProjectAdvertisingUpdate.as_view(),
         name='projects_advertising'),
+
+    url(r'^import/manual/send_mail/$',
```

Comment:
Not sure if this should live on the project import page.

Reply:
But I do like this _hint_ when the project name is already taken

Reply:
Can you suggest something if we do not want this to stay on project import page ?

Reply:
Maybe showing this on the project page that is abandoned (only for logged users, of course). And a link to that project on the import page if the name is already taken by an abandoned project. Just a personal suggestion anyway, I'm don't know if the core maintainers want to go in that direction here.

Reply:
Also some tests for the `is_abandoned` property would be awesome.

Reply:
Hi @stsewd , Shall I make the changes for displaying the option of sending mail as suggested by you ?
Also can you please elaborate on the tests which will help reduce false results on the `is_abandoned` property.
Thanks

Reply:
From what Eric said

> I think we should keep it light weight in terms of implementation so this isn't a lot to maintain. Perhaps just an admin-only button that says "email project owner with our abandoned project email" as a start, which sends an email from a template that we have in the repo.

I think this button must be only shown to admins (rtd admins) and probably should be on the project page.

About the tests, you can create a project with different scenarios like failed build but not too old, too old but with successful build, etc. Probably the tests should live on this file https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/rtd_tests/tests/test_project.py

Reply:
Okay Got it !
Thank you